src/uwtools/tests/drivers/test_fv3.py

```diff
@@ -45,64 +44,64 @@ def config(tmp_path):
 
 @fixture
 def config_file(config, tmp_path):
-    path = tmp_path / "fv3.yaml"
+    path = tmp_path / "config.yaml"
     with open(path, "w", encoding="utf-8") as f:
         yaml.dump(config, f)
     return path
 
 
 @fixture
-def fv3obj(config_file, cycle):
+def driverobj(config_file, cycle):
```

Comment:
Here, too, since I think we'll be doing some copy-paste-tweak in the future, I tried to make names a little more generic so that we'll need to edit them less when we borrow from existing code.